browser/resources/settings/email_aliases_page/email_aliases_page_index.html

```diff
@@ -0,0 +1,9 @@
+<style include="settings-shared"></style>
+
+<div class="cr-centered-card-container">
+  <settings-subpage page-title="Email Aliases">
```

Comment:
if email aliases is a `settings-subpage` that should be inside `settings-email-aliases-page` now

Reply:
Yes, that's now defined in `email_aliases_section.html`